status_test.go

```diff
@@ -134,7 +135,7 @@ var _ = gc.Suite(&StatusHistorySerializationSuite{})
 
 func emptyStatusHistoryMap() map[interface{}]interface{} {
 	return map[interface{}]interface{}{
-		"version": 1,
+		"version": 2,
```

Comment:
Maybe add a test that a v1 status map with never-set=true does the right thing (ignores it and uses never-set=false)?